k8s/src/test/scala/io/buoyant/k8s/WatchableTest.scala

```diff
@@ -1,24 +0,0 @@
-package io.buoyant.k8s
-
-import com.twitter.concurrent.AsyncStream
-import io.buoyant.test.FunSuite
-import org.scalatest.prop.GeneratorDrivenPropertyChecks
-
-class WatchableTest extends FunSuite
```

Comment:
Since `IncreasingOnlyAsyncStream` was removed, I'd like to see another test somewhere that asserts that the events on `Watchable.Activity` have monotonically-increasing resource versions even if the Kubernetes API sends events out of order...

Reply:
Good catch!